### DISCUSSION TITLE
Crash Reporter

### DISCUSSION THREAD

**Discussion by shuvashish76** [JunkFood02/Seal#81] [Ideas]
If necessary can we add our own custom crash reporter as mentioned by AppManager (AM) dev here: https://github.com/MuntashirAkon/AppManager/issues/289#issuecomment-782571555

AM consider [ACRA](https://www.appbrain.com/stats/libraries/details/acra/acra) as Tracker since it [phone home by default](https://www.acra.ch/docs/Interactions).

**Comment by AxeldeWater** [comment]
If this is being added, don't forget adding an option to turn this off.

**Comment by JunkFood02** [comment]
Currently, we're not likely to handle a lot of crashes and errors. If there're reproducible `Exception` and `Error` occurring, they'll be copied to the clipboard when the 'Detailed output' option is on.
